src/helpers/mdObjBucketHelper.ts

```diff
@@ -27,3 +28,15 @@ export function getSecondaryMeasuresLocalIdentifiers(buckets: VisualizationObjec
         get(item, "measure.localIdentifier"),
     );
 }
+
+export function getMeasuresFromMdObject(
+    mdObject: VisualizationObject.IVisualizationObjectContent,
+): VisualizationObject.BucketItem[] {
+    return get(mdObject, "buckets").reduce((acc, bucket) => {
+        const measureItems: VisualizationObject.BucketItem[] = get(bucket, "items", []).filter(
+            (item: VisualizationObject.BucketItem): boolean =>
+                has(item, "measure.definition.measureDefinition"),
```

Comment:
i prefer use type guard instead of `has`.
please see https://github.com/gooddata/gooddata-typings/blob/master/src/VisualizationObject.ts#L198